<!doctype html>
<html class="no-js" lang="en">
<head>
<meta charset="utf-8" />
<meta name="viewport" content="width=device-width, initial-scale=1.0" />
<meta name="description" content="OA Text is an independent open-access scientific publisher showcases innovative research and ideas aimed at improving health by linking research and practice to the benefit of society." />
<meta name="keywords" content="OA Text, OAT, Open Access Text, OAtext, OATEXT oatext, oat, open access text" />
<link href='http://fonts.googleapis.com/css?family=Open+Sans:400,300,600,400italic,600italic,700' rel='stylesheet' type='text/css'>
<link rel="stylesheet" href="css/component.css" />
<link rel="stylesheet" href="css/foundation.css" />
<link rel="stylesheet" href="css/custom.css" />
<script src="js/vendor/modernizr.js"></script>
</head>
<body>
<?php include("header.php"); ?>
<!--banner -->
<div class="inner-banner">
  <div class="row">
    <div class="medium-12 text-center columns">
      <h1>Take a look at the Recent articles</h1>
    </div>
  </div>
</div>
<!--banner x -->
<!--CONTENT -->
<div class="inner-page">
  <div class="row">
    <div class="large-8 columns">
      <div class="inner-left">
        <h2 class="w600">Spontaneous differentiation of buffalo (Bubalus bubalis) embryonic stem cells towards germ cell lineage</h2>
        <div data-dropdown="a1" data-options="is_hover:false" class="custom-tootip-head">
          <p class="pt5 mb10 f14 author-title"><i class="fa  fa-dot-circle-o pr5 small-font text-lite-green"></i>&nbsp;Syed Mohmad Shah<sup>#,*</sup>,</p>
        </div>
        
        

        <div class="mt25"></div>
        <ul class="accordion-list w">
          <li><a href="imm-articles.php" class="active" >Article</a></li>
          <li><a href="imm-article-info.php">Article info</a></li>
          <li><a href="imm-author-info.php">Author Info</a></li>
          <li><a href="imm-figures.php">Figures & Data</a></li>
        </ul>
        <div class="clear"></div>
        <hr class="mt0"/>
                
		<h2 id="button_1">Abstract</h2>
        <p>
Buffalo ES cells were subjected to differentiation in 15% KoSR- and 15% FBS- based spontaneous differentiation media in static suspension cultures. The embryoid bodies (EBs) so formed, were analyzed for germline-specific gene expression on day 4, 8 and 14 in order to identify the optimum differentiation strategy. Immunocytochemical analysis was performed for detection of germ lineage specific markers in EBs differentiated under the identified optimum conditions. Global DNA methylation analysis was performed to assay methylation erasure in the optimized differentiation cultures. We observed a significantly (p&lt;0.05) increased expression of all germ lineage genes like <i>DAZL, VASA</i> and <i>PLZF</i> (PGC-markers); <i>SYCP</i>3, <i>PRM</i>2, <i>TNP1</i>/2 and <i>MLH</i>1 (Meiotic markers); <i>BOULE</i> and <i>TEKT</i>1 (Spermatocyte-markers); <i>GDF</i>9 and <i>ZP</i>2/3 (Oocyte-markers) upon spontaneous differentiation in comparison to undifferentiated ES cells. EBs collected from FBS-based medium showed significantly (p&lt;0.05) higher expression of early germ lineage genes (<i>DAZL</i>, <i>TNP</i>1, <i>PRM</i>2), while those from KoSR-based medium showed greater expression of meiotic and developmental genes (<i>SYCP</i>3, <i>MLH</i>1, <i>TEKT</i>1, <i>GDF</i>9 and <i>ZP</i>2). Immunocytochemistry revealed expression of c-Kit, Dazl, Vasa (PGC-markers); Sycp3, Mlh1, Protamine1 (Meiotic markers); Acrosin and Haprin (Spermatocyte-makers); and Gdf9 and Zp4 (Oocyte-markers) in day 14 EBs collected from both the cultures. However, only PGC-specific markers were detected in 14 day monolayer cultures differentiated in KoSR-based media. The levels of 5-methyl-2-deoxycytidine were not significantly (p&lt;0.05) different between EBs collected from the two media formulations. However, EBs collected from KoSR-based medium showed lower concentration of 5-Methyl-2-deoxycytidine, indicating comparatively greater methylation erasure.
</p>
		
        <h2 id="button_2">Key words:</h2>
        <p><i> buffalo embryonic stem cells, spontaneous differentiation, germ lineage, gene expression, immunocytochemistry, methylation analysis</i></p>
        
        <h2 id="button_3">Introduction</h2>
        <p>Embryonic stem (ES) cells possess the capability to differentiate into all the three germ layers (ectoderm, mesoderm and endoderm). This property makes them suitable for elucidation of molecular mechanisms underlying differentiation and embryonic development. Among all the cell types ES cells can differentiate into, primordial germ cells (PGCs) are the most fascinating as they represent <i>in vitro</i> completion of reproductive cycle of the organism from which the ES cell line is derived. Given such vital functions of germ cells for propagation of sexually reproducing organisms, their formation must be rigidly programmed which is indeed the case in lower organisms like <i>Drosophila melanogaster, Xenopus laevis, Danio rerio</i> and in various egg-laying species [1-3]. Unlike, the pre-determined model of germ cell specification in lower organisms, mammalian primordial germ cell formation occurs later in embryonic development under the guidance of extracellular signals and in a definite cellular environment of the indifferent gonad [4] where a subset of epiblast cells begins to express <i>BLIMP1</i> and <i>FRAGILIS</i> genes, thereby becoming primordial germ cells. These cells suppress <i>HOX</i> gene expression and set themselves apart from somatic differentiation programme [5]. Under the influence of various genes like <i>STELLA, c-KIT, DAZL</i> and <i>VASA,</i> PGCs migrate through genital ridge to colonize the bipotent gonad which provides necessary microenvironment for their differentiation into gametes [6-8]. The haploidy of germ cells is achieved by the delicate process of meiosis, mediated by various proteins like Vasa, Sycp, Mlh1, Transition proteins (TNPs) and Protamines (PRMs), etc. [9,10]. Further development into mature and functional gametes occurs at puberty under the influence of mature germ cell specific proteins like Gdf9 and Zona pellucida (Zp4) (in oocyte) and Tektin, Acrosin and Haprin (in spermatocyte), etc. [11-14].<br>
<br>
Since ES cells have the intrinsic capability to differentiate into any cell type; we investigated differentiation of buffalo ES cells into germ lineage cells under spontaneous differentiation in both monolayer adherent cultures as well as in floating cultures through formation of three-dimensional aggregates known as embryoid bodies (EB). The differentiation was assessed through detection and quantitation of RNA and protein markers associated with: i) PGC ii) Meiosis; iii) Spermatocyte; and iv) Oocyte. The understanding of germ line development is of crucial importance in farm animals, especially in poor breeding bubaline species so as to understand the genetics and biochemistry of this intricate process. The understanding is further obscured due to paucity of primary tissue (first trimester fetal gonads) and complexities of <i>in vivo</i> system like cellular environment and embryo manipulation [15]. The development of culture system for differentiation of species-specific embryonic stem cells into germ cells would not only abreast us with the knowledge and understanding of this process but would further help us to design the biotechnological, chemical and/or nutritional interventions for enhancing the reproductive performance of farm animals in general and bubaline species in particular. The study would bear a future application in reducing the age of sexual maturity,  increasing the gamete quantity and quality, especially of oocytes whose reservoir is thought to be fixed and much lesser in buffalo as compared to cattle, development of designer gametes, transgenic animal production as well as conservation of endangered animal species.</p>
        
        <h2 id="button_4">Materials and methods</h2>
        <h4>Ethical approval</h4>
        <p>The study was approved by Institute's Animal Ethics Committee as well as by Department of Biotechnology, New Delhi, for use of animal tissues as well as for development of the embryonic stem cell lines and further subsequent studies.</p>
        <h4>Chemicals</h4>
        <p>All chemicals and media used in the present study were obtained from Sigma Aldrich (St. Louis, MO) and the plastic ware from Falcon (Paignton, UK), unless stated otherwise.</p>
        <h4>Embryonic stem cell establishment and maintenance</h4>
        <p>Three previously established and characterized  buffalo ES cell lines developed from <i>in vitro</i> fertilization (IVF) derived  blastocysts were used in the study [16]. The primary ES cell colonies were developed  from mechanically isolated inner cell masses of day 7 blastocysts. The primary  colonies were propagated on fetal fibroblast feeder layers and maintained in  embryonic stem cell culture medium, composed of KoDMEM + 15% Knock out serum  replacer (KoSR), supplemented with 2 mM L-glutamine, 5 ng/ml bFGF (basic  fibroblast growth factor), 1000 U/ml rmLIF (recombinant murine lukaemia  inhibitory factor), 1X nonessential amino acids and 50 mg/ml gentamicin  sulphate, and incubated at 38ÂºC in a 5% CO2 incubator. The colonies  were characterized for protein and/or RNA markers associated with pluripotency  and self-renewal like <i>OCT4, NANAOG, SOX2, STAT3, REX1, FOXD3, SSEA-1, SSEA-4,  TRA-1-60, TRA-1-81, CD90, c-MYC </i>and <i>TELOMERASE. </i>The cell lines were  also characterized for Alkaline phosphatase activity, stable karyotype and  differentiation potential to three germ layers (ectoderm, mesoderm and  endoderm). The cell lines were subjected to spontaneous differentiation at  passage 20-30.</p>
        <h4>Spontaneous differentiation into germ lineage cells</h4>
        <p><b>Differentiation in floating cultures <i>via</i> EB formation:</b>  ES cell colonies were subjected to differentiation by static suspension culture (SSC) strategy which was earlier shown to have more potential for inducing differentiation across all the three germ layers than the hanging drop (HD) strategy [16]. ES cell colonies were allowed to differentiate in low attachment 35 mm Petri dishes in spontaneous differentiation medium, composed of KoDMEM supplemented with 2 mM L-glutamine, 1X nonessential amino acids and 50 Âµg/ml gentamicin sulphate. The medium was supplemented either with 15% Knock out serum replacer (KoSR) or with 15% fetal bovine serum (FBS). Half of the culture medium was replaced with fresh medium every alternate day. EBs developed in the two media formulations were analyzed by qPCR for germ lineage gene induction at day 4, 8 and 14 of culture period. EBs were also used for immunocytochemical analysis for germ line specific protein expression as well as for Global DNA methylation to assay methylation erasure.<br>
<br>
<b>Differentiation in monolayer adherent cultures:</b> ES cell colonies were cut into small clumps in 1:8 split ratios. The clumps were washed thrice with DPBS-- and treated with Trypsin-EDTA for 2-3 min to remove any attached feeder cells. This was followed by two washes in DPBS--, treatment with Accutase and repetitive pipetting, with a finely drawn glass pipette, till all the clumps dissociated into single cells. The cell pellet, obtained by centrifugation at 2000 rpm for 5 min, was seeded onto MaxGel (1:200) coated 4-well dishes for spontaneous differentiation, as described above.<br>
<br>
<b>qPCR analysis for  germ lineage gene induction: </b>Total RNA was extracted from 100 EBs, collected  randomly from the differentiation cultures, on day 4, 8 and 14 using RNeasy RNA  extraction kit (Qiagen, Germany) and employed for cDNA synthesis using  SuperScript III first strand cDNA synthesis kit (Invitrogen, USA). qPCR  analysis was performed for quantification of such genes associated with: i)  PGC-specification and proliferation (<i>OCT</i>4, <i>NANOG</i>, <i>DAZL</i> and <i>VASA</i>); ii) Meiosis (<i>MLH</i>1, <i>SYCP</i>3, <i>TNP</i>1/2 and <i>PRM</i>2; iii) Spermatocyte (<i>BOULE</i> and <i>TEKT</i>1);  and iv) Oocyte (G<i>DF</i>9  and <i>ZP</i>2  and 3). The primers and their annealing temperatures have been previously  mentioned [17]. The optimized qRT-PCR reactions contained 2 &mu;L (100ng) cDNA  template, 10 &mu;L SYBR Green PCR Master Mix Buffer (2X), and 10 pmol each of  forward and reverse primers in a total volume of 20 &micro;L. The reactions were  performed on CFX96 instrument (BioRad Hercules, CA, USA). <i>GAPDH</i> and &beta;-<i>ACTIN</i> were used as the  endogenous controls. The thermal cycling parameters were: initial denaturation  at 95&deg;C for 10 min; followed by 40 PCR cycles (denaturation: 95&deg;C for 30s,  annealing at XËC (respective for each gene as provided already; [17]. for 30s,  and extension at 72&deg;C for 30s. Following amplification, a dissociation protocol  was performed to provide evidence for a single reaction product. A no-template  control (NTC) of nuclease-free water was included in each run and all PCR  reactions were performed in triplicates. The PCR efficiencies (E) were  evaluated by a 2-fold dilution series of cDNAs (1 - 1:64 dilution) for each  pair of primers including the reference genes (<i>GAPDH</i> and <i>&beta;-ACTIN</i>)  using a cDNA pool, obtained from all the EBs, to ensure that no inhibitory  component was present in the samples. A graph of threshold cycle (Ct) versus  relative log2 copy number of the calibration sample from the  dilution series was produced and the reaction<strong> </strong>efficiency was determined for each primer set by using the slope of this  graph (E=2 (-1/slope)).  The percent efficiency of each primer pair was found to be higher than  98% and no inhibitory effect was observed in the samples. The expression of  each target gene was normalized against the geometric mean of corresponding  threshold cycles of the endogenous controls (<i>GAPDH</i> and <i>&beta;-ACTIN</i>). The relative  expressions of germ lineage associated genes were calculated using 2-&Delta;&Delta;Ct method, taking the corresponding values  for undifferentiated ES cells as the calibrator or control values [18].<br>
<br>
<b>Immunocytochemical  analysis for germ lineage proteins: </b>A total of 300 EBs,  collected randomly from both the differentiation cultures at day 14, were  subjected to immunodetection of germ lineage markers; like c-Kit, Dazl, Vasa  (PGC-specific proteins), Mlh1, Sycp3, Protamine1 (Meiotic proteins), Acrosin  and Haprin (spermatocyte-specific proteins) and Gdf9 and Zp4 (oocyte-specific  proteins). Immunocytochemical analysis was also performed on day 14 monolayer  differentiated cultures, as per the already discussed protocol [17,19]. All the  primary as well as secondary antibodies have been detailed previously [17]. The  primary antibodies were diluted 1: 100 in blocking buffer (4% normal goat serum  in DPBS) except for Dazl which was diluted at 1:50 ratio. The testicular and  ovarian tissue sections were used as positive controls while undifferentiated  buffalo ES cells were used as the negative controls. EBs were washed twice in  DPBS, followed by fixation in 4% para formaldehyde (PFA) for 20 min. After two  washes in DPBS, the EBs were treated with 0.1% Triton X-100 in DPBS for 15 min,  followed by two washes again with DPBS and incubation for 1h in blocking  solution at room temperature. The colonies were exposed to primary antibodies  at 4Â°C overnight. After three thorough washings with DPBS, the EBs were  incubated for 1h at room temperature with the appropriate class specific  secondary antibody, labeled either with fluorescein isothiocyanate (FITC) or  phycoerythrin (PE). After washing thrice with DPBS, EBs were put individually  on a clean grease free microscopic glass slide with a narrowly drawn glass  pipette and overlaid with Prolong Gold Antifade reagent with DAPI  (4-6-Diamidino-2-phenylindole) (Life technologies, USA) to stain the nuclei.  After 5-10 min, the colonies were covered with clean grease free microscopic  cover slips followed by visualization under a fluorescence microscope (Diaphot;  Nikon, Tokyo, Japan). The similar protocol was followed for immunostaining of  45 different monolayer adherent cultures differentiated for a period of 14  days.<br>
<br>
<b>Global DNA  methylation analysis: </b>Global DNA methylation analysis of  undifferentiated control ES cell colonies as well as of day 14 EBs was  performed by employing Global DNA Methylation ELISA Kit (Cell Biolabs, USA),  following the previously discussed protocol [17,19]. Genomic DNA (gDNA) was extracted by NucleoSpin Tissue DNA purification  kit (Macherey-Nagel, Germany), following the manufacturer's instructions.  Extracted gDNA samples were digested to nucleosides by treatment with nuclease  P1 and alkaline phosphatase, as described [17]. Reaction mixture was  centrifuged at 6000g for 5 min and supernatant, containing individual  nucleosides, was used for 5MedCyd ELISA assay. 50 &mu;l of sample DNA or 5MedCyd  standard was added to the wells of the 5MedCyd-DNA conjugate coated plate and  incubated at room temperature for 10 min on an orbital shaker. This was  followed by addition of pre-diluted primary antibody and incubation for 2 h on  an orbital shaker, washing with 1X wash buffer and addition of 1X blocking  reagent. After 1h incubation, secondary antibody enzyme conjugate was added to  each well followed by three washings and addition of substrate solution. Enzyme  reaction was stopped by addition of 100 &mu;l stop solution as soon as the color  started to appear. The results were read immediately on spectrophotometer using  450 nm as primary wavelength and standard curve was prepared using standard  5-methyl-2-deoxycytidine concentrations <i>versus </i>corresponding absorbance values. The  absorbance values of unknown samples were subsequently used to calculate the  concentration of 5-Methyl-2-deoxycytidine within their gDNAs. The samples were  analyzed for 5-methyl-2-deoxycytidine concentrations based on mean absorbance  values of three independent measurements, each taken in duplicate.
</p>
   
   <h4>Statistical analysis</h4>
   <p>qPCR data for KoSR vs FBS based differentiation media  were compared by Student's t-test using a statistical software program (SPSS 11.5, 2004, IBM, USA). GraphPad PRISM version 4 (GraphPad Software Inc., San Diego CA) was used for building standard curve and calculation of 5-methyl-2-deoxycytidine concentrations directly from the corresponding absorption values. The results are expressed as mean Â± S.E. of mean and the statistical significance was accepted at p &lt;0.05.</p>
   <h2 id="button_5">Results</h2>
   <h4>qPCR analysis for germ lineage genes</h4>
   <p>qPCR expression profiles are shown in Figure 1  for both KoSR and FBS generated EBs, taking undifferentiated ES cells as the  calibrator. <i>NANOG</i> showed decrease in expression throughout the culture period (normalized  expression less than 1) as compared to the calibrator (normalized expression  equal to 1) under both the differentiation conditions. <i>OCT</i>4 showed a six-fold increased  expression in day 8 EBs produced in KoSR differentiation medium while the  corresponding expression in FBS based medium was lesser than the control. <i>DAZL</i> showed significantly (p&lt;0.05) increased expression (2.5 fold greater than  control) in day 14 EBs obtained from FBS based spontaneous differentiation  medium, while the corresponding expression in day 4 and day 8 EBs  revealed no significant difference between KoSR and FBS based media. <i>VASA</i> showed significantly increased expression (3-fold higher) in day 4 EBs obtained  in KoSR based medium as compared to the corresponding EBs from FBS based  medium. Its expression although greater than control, showed no significant  (p&lt;0.05) difference in day 8 and day 14 EBs, differentiated in KoSR and FBS  based media. <i>PLZF</i> did not show any significant change in expression pattern throughout the  culture interval in both types of differentiation media. <i>SYCP</i>3 showed significantly  greater expression in KoSR-based media, throughout the culture interval, with  the highest expression in day 8 EBs.Â  <i>MLH</i>1  showed highest expression in day 4 EBs obtained from FBS based medium,  thereafter showing a decreasing trend as the culture interval prolonged. Its  expression was significantly greater in day 8 EBs obtained from KoSR based  SSCs, while day 14 EBs did not show any significant (p&lt;0.05) difference in  expression profile between the two media types. <i>PRM</i>2 expression was highest in  day 8 EBs obtained from both the media formulations in comparison to the  corresponding day 4 and day 14 EBs, with the highest expression in EBs produced  in FBS based medium. The expression of <i>BOULE</i> was significantly (p&lt;0.0) greater in  FBS based media throughout the culture interval, while <i>TEKT</i>1expression showed a significant  increase in day 14 EBs obtained from KoSR supplemented medium. Day 4 EBs  derived from FBS based medium showed a significantly (p&lt;0.05) greater  expression of <i>GDF</i>9,  unlike day 8 EBs which showed significantly increased expression in KoSR based  media. 14 day old EBs did not show any significant (p&lt;0.05) difference in  expression pattern of <i>GDF</i>9 between the two media types. <i>ZP</i>2expression  was highest (2-3 fold) in day 8 EBs as compared to the corresponding day 4 and  day 14 ones. The expression, however, was not significantly different between  the two media types while <i>ZP</i>3 showed increased expression, though not  significant, in KoSR-based EBs analyzed at day 4 and 14 of the culture (Figure 1). Hence, it could be concluded that ES cells  do express various germ linage associated markers upon spontaneous  differentiation.</p>
   <img src="103_img/JSCR-1-103-g001.gif"><br>
   
   <h4>Immunocytochemical analysis for germ lineage proteins</h4>
   <p>Based on qPCR data analysis, immunocytochemical  examination of various germ lineage associated proteins was performed on both  day 14 EBs as well as monolayer adherent cultures. Immunocytochemistry revealed  expression of: i) PGC markers like c-Kit, Dazl and Vasa (Figure 2); ii) Meiotic proteins like Sycp3, Mlh1 and  Protamine1 (Figure 3); iii) Spermatocyte markers  like Acrosin and Haprin (Figure 4); and iv)  Oocyte specific markers like Gdf9 and Zp4 (Figure 5)  in EBs collected from both KoSR and FBS-based media formulations. The number of  EBs that stained positive for various germ lineage associated proteins is given  in table 1. Only 3 monolayer adherent cultures revealed the positive expression  of PGC-specific proteins out of the 45 cultures that were analyzed (Figure 6). None of the cultures, both from  KoSR-and FBS-based spontaneous differentiation cultures, showed the presence of meiotic and mature germ cell proteins upon  immunocytochemical analysis. No such protein detection was observed in  undifferentiated (control) ES colonies and monolayer cultures (Figure S1 and S2)</p>
   
   
   <p><b>Table 1.</b> Immunocytochemical analysis results for germ lineage gene induction in   embryoid bodies derived from KoSR- and FBS-based spontaneous differentiation media.</p>
   <table border="1" cellspacing="0" cellpadding="0">

  <tr>
    <td align="center" colspan="11"><b>KoSR-based spontaneous differentiation</b></td>


 </tr>
 <tr>
    <td>Germ lineage protein</td>
    <td>c-KIT</td>
    <td>DAZL</td>
    <td>VASA</td>
    <td>MLH1</td>
    <td>SYCP3</td>
    <td>PROTAMINE1</td>
    <td>ACROSI</td>
    <td>HAPRIN</td>
    <td>GDF9</td>
    <td>ZP4</td>
 </tr>
 <tr>
    <td>Total EBs analyzed</td>
    <td align="left">24</td>
   <td align="left">24</td>
   <td align="left">24</td>
   <td align="left">30</td>
   <td align="left">30</td>
   <td align="left">30</td>
   <td align="left">35</td>
   <td align="left">35</td>
   <td align="left">34</td>
   <td align="left">34</td>
 </tr>
 
  <tr>
    <td>EBs expressing the protein</td>
   <td align="left">6<br>(25%)</td>
   <td align="left">7<br>(29.2%)</td>
   <td align="left">8<br>(33.3%)</td>
   <td align="left">4<br>(13.3%)</td>
   <td align="left">3<br>(10%)</td>
   <td align="left">3<br>(10%)</td>
   <td align="left">3<br>(8.6%)</td>
   <td align="left">4<br>(11.4%)</td>
   <td align="left">5<br>(14.7%)</td>
   <td align="left">4<br>(11.8%)</td>
 </tr>
 <tr>
    <td colspan="11" align="center"><b>FBS-based spontaneous differentiation</b></td>
 </tr>
 <tr>
    <td>Germ lineage protein</td>
    <td>c-KIT</td>
    <td>DAZL</td>
    <td>VASA</td>
    <td>MLH1</td>
    <td>SYCP3</td>
    <td>PROTAMINE1</td>
    <td>ACROSI</td>
    <td>HAPRIN</td>
    <td>GDF9</td>
    <td>ZP4</td>
 </tr>
 <tr>
    <td>Total EBs analyzed</td>
   <td align="left">24</td>
   <td align="left">24</td>
   <td align="left">24</td>
   <td align="left">30</td>
   <td align="left">30</td>
   <td align="left">30</td>
   <td align="left">35</td>
   <td align="left">35</td>
   <td align="left">34</td>
   <td align="left">34</td>
 </tr>
 <tr>
    <td>EBs expressing the protein</td>
   <td align="left">6<br>(25%)</td>
   <td align="left">6<br>(25%)</td>
   <td align="left">6<br>(25%)</td>
   <td align="left">3<br>(10%)</td>
   <td align="left">2<br>(6.7%)</td>
   <td align="left">2<br>(6.7%)</td>
   <td align="left">2<br>(5.7%)</td>
   <td align="left">1<br>(2.8%)</td>
   <td align="left">3<br>(8.8%)</td>
   <td align="left">2<br>(5.9%);</td>
 </tr>
 
 
</table>

          <center>
 <img src="103_img/JSCR-1-103-g002.gif"><br>
                      <img src="103_img/JSCR-1-103-g003.gif"><br>
                                 <img src="103_img/JSCR-1-103-g004.gif"><br>
                                            <img src="103_img/JSCR-1-103-g005.gif"><br>
                                                       <img src="103_img/JSCR-1-103-g006.gif"><br>
                                                       <img src="103_img/JSCR-1-103-g007.gif"><br>
</center>
        
        <h2 id="button_6">Discussion</h2>
        <h4>Assaying ES cell differentiation to germ lineage</h4>
        <p>A fairly good number of reports exist which  demonstrate that ES cells, irrespective of the species of origin, are  pluripotent and capable of differentiating into a wide variety of cell types  under appropriate culture conditions [9,20]. However, very scanty information  is available on differentiation of embryonic stem cells into germ cells, apart  from some reports in mice and human embryonic and/or induced pluripotent stem  cells [8,9,11,21,22]. However, germ lineage differentiation from bubaline ES  cells or stem cell-like cells has not been reported and, as per our knowledge,  this is the first draft reporting spontaneous differentiation of bubaline  embryonic stem cells into germ lineage cells. For this purpose, we studied a  panel of germ cell-specific/ enriched markers together with embryonic stem cell  markers to model <i>in  vitro</i> differentiation of ES cells into germ lineage cells. There are  a number of evidences which support the validity of this approach to assay germ  cell development [9,23,24].Â  A belief,  though untrue, was held that ES cells express all the genes promiscuously thus  annulling cell-specific marker analysis as inappropriate means to monitor their  differentiation into any given cell type. However, a number of reports have demonstrated  that ES cells do not express all genes in a promiscuous manner [25-29], thereby  validating the marker-based identification approach. We also observed a  non-random pattern of gene expression in our study which began with expression  of early markers of germ line-specific/ enriched genes (PGC-specific markers)  at lesser culture intervals, while markers associated with meiosis and  spermatocyte as well as oocyte development appeared to show appreciable  expression at later stages of culture interval. We also connected transcription  of germ cell-specific genes with translation. The reports that only germ line  cells express proteins like Dazl, Vasa, Sycp3, Gdf9, Tekt1, Tnps, Prms,  Acrosin, Boule, Zp<sup>4</sup>, Haprin etc. [9,11,14,22] further support our model to  investigate ES cell differentiation into germ lineage cells.</p>

        <h4>Transcriptional profiling of germ lineage genes</h4>
        <p>It has been reported  that ES cells are reprogrammed spontaneously to go through germ lineage, in  addition to their differentiation into many other lineages. Such reprogramming  has been reported in mice when murine ES cells were cultured in absence of  pluripotency maintaining factors like rmLIF and feeder cells [23,24,30,31].  Spontaneous differentiation to germ line cells has been reported in murine as  well as human ES cells in adherent cultures after removal of pluripotency  factors like rmLIF and bFGF [32,33] as well as through formation of embryoid  bodies [30,31,34]. Upon spontaneous differentiation through EB culture, we  observed significantly (p&lt;0.05) increased expression of germ lineage  associated genes in EBs collected at 4, 8 and 14 days of culture interval as  compared to the control ES cell colonies. The highest expression of the genes  was more or less observed in 8 day old EBs in both types of differentiation  cultures. FBS based spontaneous differentiation cultures induced greater  expression of genes like <i>DAZL</i>, <i>TNP</i>1, <i>PRM</i>2 and <i>BOULE</i> as compared to KoSR-based spontaneous  cultures, probably, due to presence of a large number of growth factors present  in FBS as compared to KoSR, which is a commercial serum replacer, designed  primarily for maintaining stem cells in an undifferentiated state under normal  culture conditions. The active ingredient in serum replacer is a lipid- rich  albumin which, unlike albumin of FBS, has a strong positive effect on the  expansion of undifferentiated human ES cells in culture [35]. The chemical  nature of FBS owing to its defined and/or undefined components could also lead  to random and non-tailored differentiation of the pluripotent cells across  different lineages.</p>
        <h4>Immuocytochemical analysis for germ lineage markers</h4>
        <p>Based on the germ lineage gene expression data from qPCR, we collected day 14 EBs for immunocytochemistry.  These EBs expressed PGC-specific markers (c-Kit, Dazl and Vasa); Meiotic markers (Sycp3, Mlh1 and Protamine); Spermatocyte-specific markers (Acrosin and Haprin); as well as Oocyte-specific markers (Gdf9 and Zp4); indicating differentiation into germ lineage-like cells. However, MaxGel-adherent differentiated monolayer showed only expression of PGC-specific markers like c-Kit, Dazl and Vasa, indicating that the differentiation has not progressed into meiotic and later germ lineage events. Since, germ cell differentiation process is dependent on spatial distribution of differentiating cells, the three-dimensional structure as represented by EBs probably provide a microenvironment that enhances germ lineage differentiation, although other cell types besides germ cells may also arise from them [36]. As per this observation, EB culture provides a better alternative than adherent culture for inducing germ lineage differentiation but the most pertinent problem with EB culture is sorting of the differentiated cells from EBs, since they grow as tight three-dimensional aggregates, unlike monolayer cultures which could be readily disaggregated by enzymatic treatment.</p>
        <h4>Global DNA methylation analysis</h4>
        <p>It is a well-known fact that epigenetic modifications are sequentially established and erased in germ lineage. PGCs undergo DNA demethylation of the imprinted loci to erase the parental imprints [37,38]. The imprinting process is mainly achieved by demethylation of the cytosine residues in CpG islands. It has been reported that ES cells retain the ability to carry out global <i>de novo</i> methylation unlike their somatic counterparts, but at the same time are capable of distinguishing and protecting CpG islands from this process [39], thereby ensuring their self-renewal and unspecialized (undifferentiated) nature. Though 5-methyl-2-deoxycytidine concentration was not significantly different in undifferentiated ES cells and day 14 EBs collected from KoSR- and FBS-based spontaneous differentiation cultures, the concentration was nevertheless lower in EBs collected from KoSR based spontaneous cultures, indicating comparatively  greater methylation erasure.<br>
<br>
To our knowledge, this is the first study of its  type in farm animals aiming at differentiation of bubaline ES cells into germ  line cells under spontaneous differentiation <i>milieu</i>. We observed that KoSR-  based spontaneous medium would be better in inducing differentiation to germ  lineage than FBS- based medium which contains a host of undefined chemical  components that might hinder progression into meiosis and post-meiotic germ  cell development. The addition of exogenous factors that are capable of  inducing germ lineage differentiation should, hence, be attempted in KoSR-based  medium and not in FBS-based medium. EB culture also revealed to be more  efficient in inducing differentiation than monolayer culture but the problem  lies in isolation and sorting of the differentiated cells from these  three-dimensional aggregates. The obvious solution would be to attempt  3-dimensional culture strategies for ES cell monolayer culture which would combine  both the advantages of EB culture and the ease of sorting for differentiated  cells. It could be concluded that ES cells can be differentiated into germ  lineage cells under <i>in vitro</i> culture and there is a sound  possibility of generating the mature and functionally competent germ cells <i>in vitro</i>.  Such studies will also enhance our knowledge of farm animal gametogenesis  especially of bubaline species. The future implications would be to develop the  strategies, whether nutritional, managerial, chemical or biotechnological,  which would enhance the quality and quantity of the gametes, reduce the age of  sexual maturity which currently goes upto 2-3 years in buffalo, and develop and  design the gametes of choice for transgenic buffalo production and conservation  of endangered species by development of artificial gametes.</p>
<h2 id="button_7">Conflict of interest</h2>
<p>The authors declare no conflict of interest.</p>
        
        <h2>References</h2>
         <ul>
        	<li>Strome S, Lehmann R (2007) Germ versus soma decisions: lessons from flies and worms. <i>Science</i> 316: 392-393.&nbsp;[<a href="http://www.ncbi.nlm.nih.gov/pubmed/17446385">Crossref</a>]</li>
 <li>Jin Z, Xie T (2006) Germline specification: small things have a big role. <i>Curr Biol</i> 16: R966-967.&nbsp;[<a href="http://www.ncbi.nlm.nih.gov/pubmed/17113380">Crossref</a>]</li>
 <li>Baughman JM, Geijsen N (2005) In vitro generation of germ cells: new techniques to solve current issues. <i>Ann N Y Acad Sci</i> 1061: 33-40.&nbsp;[<a href="http://www.ncbi.nlm.nih.gov/pubmed/16467255">Crossref</a>]</li>
 <li>Saitou M, Barton SC, Surani MA (2002) A molecular programme for the specification of germ cell fate in mice. <i>Nature</i> 418: 293-300.&nbsp;[<a href="http://www.ncbi.nlm.nih.gov/pubmed/12124616">Crossref</a>]</li>
 <li>Ancelin K, Lange UC, Hajkova P, Schneider R, Bannister AJ, et al. (2006) Blimp1 associates with Prmt5 and directs histone arginine methylation in mouse germ cells. <i>Nat Cell Biol</i> 8: 623-630.[<a href="http://www.ncbi.nlm.nih.gov/pubmed/16699504">Crossref</a>]</li>
 <li>Bortvin A, Goodheart M, Liao M, Page DC (2004) Dppa3 / Pgc7 / stella is a maternal factor and is not required for germ cell specification in mice. <i>BMC Dev Biol</i> 4: 2.&nbsp;[<a href="http://www.ncbi.nlm.nih.gov/pubmed/15018652">Crossref</a>]</li>
 <li>Nakamura T, Arai Y, Umehara H, Masuhara M, Kimura T, et al. (2007) PGC7/Stella protects against DNA demethylation in early embryogenesis. <i>Nat Cell Biol</i> 9: 64-71.&nbsp;[<a href="http://www.ncbi.nlm.nih.gov/pubmed/17143267">Crossref</a>]</li>
 <li>Geijsen N, Horoschak M, Kim K, Gribnau J, Eggan K, et al. (2004) Derivation of embryonic germ cells and male gametes from embryonic stem cells. <i>Nature</i> 427: 148-154.&nbsp;[<a href="http://www.ncbi.nlm.nih.gov/pubmed/14668819">Crossref</a>]</li>
 <li>Clark AT, Bodnar MS, Fox M, Rodriquez RT, Abeyta MJ, et al. (2004) Spontaneous differentiation of germ cells from human embryonic stem cells in vitro. <i>Hum Mol Genet</i> 13: 727-739. [<a href="http://www.ncbi.nlm.nih.gov/pubmed/14962983">Crossref</a>]</li>
 <li>Marques-Mari AI, Lacham-Kaplan O, Medrano JV, Pellicer A, SimÃ³n C (2009) Differentiation of germ cells and gametes from stem cells. <i>Hum Reprod Update</i> 15: 379-390.&nbsp;[<a href="http://www.ncbi.nlm.nih.gov/pubmed/19179344">Crossref</a>]</li>
 <li>Aflatoonian B, Ruban L, Jones M, Aflatoonian R, Fazeli A et al. (2009) In vitro post-meiotic germ cell development from human embryonic stem cells. <i>Hum Reprod</i> 24: 3150-3159. [<a href="http://www.ncbi.nlm.nih.gov/pubmed/19770126">Crossref</a>]</li>
 <li>West FD, Machacek DW, Boyd NL, Pandiyan K, Robbins KR, et al. (2008) Enrichment and differentiation of human germ-like cells mediated by feeder cells and basic fibroblast growth factor signaling. <i>Stem Cells</i> 26: 2768-2776.&nbsp;[<a href="http://www.ncbi.nlm.nih.gov/pubmed/18719225">Crossref</a>]</li>
 <li>Tilgner K, Atkinson SP, Golebiewska A, Stojkovic M, Lako M, et al. (2008) Isolation of primordial germ cells from differentiating human embryonic stem cells. <i>Stem Cells</i> 26: 3075-3085. [<a href="http://www.ncbi.nlm.nih.gov/pubmed/18802037">Crossref</a>]</li>
 <li>Shah SM, Saini N, Ashraf S, Zandi M, Singh MK, et al. (2015) Comparative Expression Analysis of Gametogenesis-Associated Genes in Foetal and Adult Bubaline (Bubalus bubalis) Ovaries and Testes. <i>Reprod Domest Anim </i>50: 365-377.&nbsp;[<a href="http://www.ncbi.nlm.nih.gov/pubmed/25703697">Crossref</a>]</li>
 <li>Wei W, Qing T, Ye X, Liu H, Zhang D, et al. (2008) Primordial germ cell specification from embryonic stem cells. <i>PLoS One </i>3: e4013.&nbsp;[<a href="http://www.ncbi.nlm.nih.gov/pubmed/19107197">Crossref</a>]</li>
 <li>Shah SM, Saini N, Ashraf S, Zandi M, Manik RS, et al. (2015) Development, Characterization, and Pluripotency Analysis of Buffalo (Bubalus bubalis) Embryonic Stem Cell Lines Derived from In Vitro-Fertilized, Hand-Guided Cloned, and Parthenogenetic Embryos. <i>Cell Reprogram </i>17: 306-322.&nbsp;[<a href="http://www.ncbi.nlm.nih.gov/pubmed/26168169">Crossref</a>]</li>
 <li>Shah SM, Saini N, Singh MK, Manik R, Singla SK, et al. (2016) Testicular cell-conditioned medium supports embryonic stem cell differentiation toward germ lineage and to spermatocyte- and oocyte-like cells. <i>Theriogenology</i>.&nbsp;[<a href="http://www.ncbi.nlm.nih.gov/pubmed/27056417">Crossref</a>]</li>
 <li>Schmittgen TD, Livak KJ (2008) Analyzing real-time PCR data by the comparative C(T) method. <i>Nat Protoc</i> 3: 1101-1108.&nbsp;[<a href="http://www.ncbi.nlm.nih.gov/pubmed/18546601">Crossref</a>]</li>
 <li>Shah SM, Saini N, Ashraf S, Singh MK, Manik RS, et al. (2015) Cumulus cell-conditioned medium supports embryonic stem cell differentiation to germ cell-like cells. <i>Reprod Fertil Dev</i>. [<a href="http://www.ncbi.nlm.nih.gov/pubmed/26595369">Crossref</a>]</li>
 <li>Odorico JS, Kaufman DS, Thomson JA (2001) Multilineage differentiation from human embryonic stem cell lines. <i>Stem Cells </i>19: 193-204.&nbsp;[<a href="http://www.ncbi.nlm.nih.gov/pubmed/11359944">Crossref</a>]</li>
 <li>Okita K, Ichisaka T, Yamanaka S (2007) Generation of germline-competent induced pluripotent stem cells. <i>Nature</i> 448: 313-317.&nbsp;[<a href="http://www.ncbi.nlm.nih.gov/pubmed/17554338">Crossref</a>]</li>
 <li>Kee K, Angeles VT, Flores M, Nguyen HN, Reijo Pera RA (2009) Human DAZL, DAZ and BOULE genes modulate primordial germ-cell and haploid gamete formation. <i>Nature</i> 462: 222-225. [<a href="http://www.ncbi.nlm.nih.gov/pubmed/19865085">Crossref</a>]</li>
 <li>HÃ¼bner K, Fuhrmann G, Christenson LK, Kehler J, Reinbold R, et al. (2003) Derivation of oocytes from mouse embryonic stem cells. <i>Science</i> 300: 1251-1256.&nbsp;[<a href="http://www.ncbi.nlm.nih.gov/pubmed/12730498">Crossref</a>]</li>
 <li>Toyooka Y, Tsunekawa N, Akura R, Noce T (2003) Embryonic stem cells can form germ cells in vitro. <i>Proc Natl Acad Sci U S A </i>100: 11457-11462. [<a href="http://www.ncbi.nlm.nih.gov/pubmed/14504407">Crossref</a>]</li>
 <li>Ivanova NB, Dimos JT, Schaniel C, Hackney JA, Moore KA, et al. (2002) A stem cell molecular signature. <i>Science</i> 298: 601-604.&nbsp;[<a href="http://www.ncbi.nlm.nih.gov/pubmed/12228721">Crossref</a>]</li>
 <li>Ramalho-Santos M, Yoon S, Matsuzaki Y, Mulligan RC, Melton DA (2002) &quot;Stemness&quot;: transcriptional profiling of embryonic and adult stem cells. <i>Science </i>298: 597-600.&nbsp;[<a href="http://www.ncbi.nlm.nih.gov/pubmed/12228720">Crossref</a>]</li>
 <li>Tanaka TS, Kunath T, Kimber WL, Jaradat SA, Stagg CA, et al. (2002) Gene expression profiling of embryo-derived stem cells reveals candidate genes associated with pluripotency and lineage specificity. <i>Genome Res</i> 12: 1921-1928.&nbsp;[<a href="http://www.ncbi.nlm.nih.gov/pubmed/12466296">Crossref</a>]</li>
 <li>Hsiao LL, Dangond F, Yoshida T, Hong R, Jensen RV, et al. (2001) A compendium of gene expression in normal human tissues. <i>Physiol Genomics</i> 7: 97-104.&nbsp;[<a href="http://www.ncbi.nlm.nih.gov/pubmed/11773596">Crossref</a>]</li>
 <li>Warrington JA, Nair A, Mahadevappa M, Tsyganskaya M (2000) Comparison of human adult and fetal expression and identification of 535 housekeeping/maintenance genes. <i>Physiol Genomics</i> 2: 143-147.&nbsp;[<a href="http://www.ncbi.nlm.nih.gov/pubmed/11015593">Crossref</a>]</li>
 <li>Chen HF, Kuo HC, Chien CL, Shun CT, Yao YL, et al. (2007) Derivation, characterization and differentiation of human embryonic stem cells: comparing serum-containing versus serum-free media and evidence of germ cell differentiation. <i>Hum Reprod</i> 22: 567-577.&nbsp;[<a href="http://www.ncbi.nlm.nih.gov/pubmed/17071820">Crossref</a>]</li>
 <li>Qing T, Shi Y, Qin H, Ye X, Wei W, et al. (2007) Induction of oocyte-like cells from mouse embryonic stem cells by co-culture with ovarian granulosa cells. <i>Differentiation</i> 75: 902-911.&nbsp;[<a href="http://www.ncbi.nlm.nih.gov/pubmed/17490416">Crossref</a>]</li>
 <li>Nayernia K, Lee JH, Drusenheimer N, Nolte J, Wulf G, et al. (2006) Derivation of male germ cells from bone marrow stem cells. <i>Lab Invest</i> 86: 654-663.&nbsp;[<a href="http://www.ncbi.nlm.nih.gov/pubmed/16652109">Crossref</a>]</li>
 <li>Novak I, Lightfoot DA, Wang H, Eriksson A, Mahdy E, et al. (2006) Mouse embryonic stem cells form follicle-like ovarian structures but do not progress through meiosis. <i>Stem Cells</i> 24: 1931â1936. [<a href="http://www.ncbi.nlm.nih.gov/pubmed/16644921">Crossref</a>]</li>
 <li>Lacham-Kaplan O, Chy H, Trounson A (2006) Testicular cell conditioned medium supports differentiation of embryonic stem cells into ovarian structures containing oocytes. <i>Stem Cells</i> 24: 266-273.&nbsp;[<a href="http://www.ncbi.nlm.nih.gov/pubmed/16109761">Crossref</a>]</li>
 <li>Garcia-Gonzalo FR, IzpisÃºa Belmonte JC (2008) Albumin-associated lipids regulate human embryonic stem cell self-renewal. <i>PLoS One</i> 3: e1384.&nbsp;[<a href="http://www.ncbi.nlm.nih.gov/pubmed/18167543">Crossref</a>]</li>
 <li>Itskovitz-Eldor J, Schuldiner M, Karsenti D, Eden A, Yanuka O, et al. (2000) Differentiation of human embryonic stem cells into embryoid bodies compromising the three embryonic germ layers. <i>Mol Med</i> 6: 88-95.&nbsp;[<a href="http://www.ncbi.nlm.nih.gov/pubmed/10859025">Crossref</a>]</li>
 <li>Lee J, Inoue K, Ono R, Ogonuki N, Kohda T, et al. (2002) Erasing genomic imprinting memory in mouse clone embryos produced from day 11.5 primordial germ cells. <i>Development</i> 129: 1807-1817.&nbsp;[<a href="http://www.ncbi.nlm.nih.gov/pubmed/11934847">Crossref</a>]</li>
 <li>Yamazaki Y, Mann MR, Lee SS, Marh J, McCarrey JR, et al. (2003) Reprogramming of primordial germ cells begins before migration into the genital ridge, making these cells inadequate donors for reproductive cloning. <i>Proc Natl Acad Sci U S A</i> 100: 12207-12212.&nbsp;[<a href="http://www.ncbi.nlm.nih.gov/pubmed/14506296">Crossref</a>]</li>
 <li>Ludwig G, Nejman D, Hecht M, Orlanski S, Abu-Remaileh M, et al. (2014) Aberrant DNA methylation in ES cells. <i>PLoS One</i> 9: e96090.&nbsp;[<a href="http://www.ncbi.nlm.nih.gov/pubmed/24852222">Crossref</a>]</li>
            </ul>
       
               
      </div>
    </div>
    <div class="large-4 columns">
      <div class="inner-right">
        <div class="row">
          <div class="small-12 columns">
            <h4 class="left mr20 mt5">Articles</h4>
            <a href="pdf/IMM-E001.pdf" target="_blank" class="left mr5"><img src="img/icon-pdf.png" alt=""></a><a href="#" class="left mr5"><img src="img/icon-xml.png" alt=""></a></div>
        </div>
        <hr/>
        <h4><a href="IntegrativeMolecularMedicine.php">Journal Home</a></h4>
        <hr/>
        <div id="sticky-anchor"></div>
        <div id="sticky">
          <h2>Jump to:</h2>
          <hr/>
          <div class="row collapse">
            <div class="large-6 columns">
              <ul class="main-bar-list">
                <li><a href="imm-articles.php" class="active">Article</a></li>
                <li><a href="imm-article-info.php" >Article Info</a></li>
                <li><a href="imm-author-info.php" >Author Info</a></li>
                <li><a href="imm-figures.php" >Figures & Data</a></li>
              </ul>
            </div>
            <div class="large-6 columns">
              <ul class="right-bar-list">
                <li id="button_1"><a href="#button_1" >Abstract</a></li>
                <li id="button_2"><a href="#button_2" >Key words</a></li>
                <li id="button_3"><a href="#button_3" >Introduction</a></li>
                <li id="button_4"><a href="#button_4" >Materials and methods</a></li>
                <li id="button_5"><a href="#button_5" >Results</a></li>
                <li id="button_6"><a href="#button_6" >Discussion</a></li>
                <li id="button_7"><a href="#button_7" >Conflict of interest</a></li>
              </ul>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
<!--CONTENT X -->
<a href="#" class="scrollToTop"><i class="fa fa-angle-up"></i></a>
<!--****************************************************** FOOTER *****************************************************-->
<div class="wrapper gray-bg footer">
  <div class="row">
    <div class="large-6 columns">
      <ul>
        <li><a href="PrivacyPolicy.php">Privacy Policy</a></li>
        <li><a href="Terms- conditions.php">Terms & conditions</a></li>
      </ul>
    </div>
    <div class="large-6 columns">
      <p class="text-right white-text">Â© 2014 Copyright OAT. All rights reserved.</p>
    </div>
  </div>
</div>
<!--****************************************************** FOOTER-X *****************************************************-->
<!--home content s -->
<script src="js/vendor/jquery.js"></script>
<script src="js/scroll.js"></script>
<script>
	$(window).scroll(function(){
		if ($(this).scrollTop() > 100) {
			$('.scrollToTop').fadeIn();
		} else {
			$('.scrollToTop').fadeOut();
		}
	});
	
	//Click event to scroll to top
	$('.scrollToTop').click(function(){
		$('html, body').animate({scrollTop : 0},800);
		return false;
	});
		</script>
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-53268443-1', 'auto');
  ga('send', 'pageview');

</script>
<script src="js/foundation.min.js"></script>
<script>
      $(document).foundation();
    </script>
</body>
</html>
